CHANGELOG.md

```diff
@@ -1775,6 +1775,7 @@ Released 2018-09-13
 [`range_plus_one`]: https://rust-lang.github.io/rust-clippy/master/index.html#range_plus_one
 [`range_step_by_zero`]: https://rust-lang.github.io/rust-clippy/master/index.html#range_step_by_zero
 [`range_zip_with_len`]: https://rust-lang.github.io/rust-clippy/master/index.html#range_zip_with_len
+[`rc_buffer`]: https://rust-lang.github.io/rust-clippy/master/index.html#rc_buffer
```

Comment:
> Also, I don't know if "buffer type" is the best way to put it or not. Alternatively I could call it a "growable type" or "growable buffer type", but I was thinking of PathBuf when I started making the lint.

I think `rc_buffer` is fine but if you're looking for alternatives you might highlight that the problem is with `Rc<SmartPointer>` instead of `Rc<<SmartPointer as Deref>::Target>`, so maybe something like `smart_pointer_in_rc`

Reply:
Hm, okay, I'll not fuss over it too much then.